package.json

```diff
@@ -18,12 +18,14 @@
     "pretest": "npm run lint",
     "prewatch:browser": "npm run clean:browser",
     "prewatch:mobile": "npm run clean:mobile",
-    "lint": "standard 'src/**/*.js' 'src/**/*.jsx' 'mobile/src/**/*.js' 'mobile/src/**/*.jsx'",
+    "lint": "npm-run-all --parallel 'lint:*'",
+    "lint:styles": "stylint src/styles --config ./.stylintrc",
+    "lint:js": "standard 'src/**/*.js' 'src/**/*.jsx' 'test/**/*.js' 'mobile/src/**/*.js' 'mobile/src/**/*.jsx' 'mobile/test/**/*.js'",
     "test": "node --harmony_proxies node_modules/.bin/jest",
     "watch": "webpack --config webpack.config.js --watch --display-modules --display-chunks",
```

Comment:
So what do you prefer? Do we need to have a shadowed task like `sub:watch` and alias `watch` to `watch:browser`? Or do we keep it like this and never use the `watch` or `build` tasks alone?